dnf5/commands/download/download.cpp

```diff
@@ -152,6 +152,9 @@ void DownloadCommand::run() {
         libdnf::repo::PackageDownloader downloader(ctx.base);
         auto destination = ctx.base.get_config().get_destdir_option().get_value().c_str();
 
+        // for download command, we don't want to mark the packages for removal
+        downloader.force_keep_packages(true);
```

Comment:
This is not a mistake, but I think it is redundand. I guess that the download command always use destdir (default value is '"."').